geth/rpc/client.go

```diff
@@ -41,6 +43,7 @@ func NewClient(client *gethrpc.Client, upstream params.UpstreamRPCConfig) (*Clie
 	c := Client{
 		local:    client,
 		handlers: make(map[string]Handler),
+		log:      log.New("package", "status-go/geth/rpc.Client"),
```

Comment:
hmm, we don't use it anywhere?

Reply:
I initially thought this too while doing my final pass :laughing:  It's actually used in `callBatchMethods` in call_raw.go